src/Microsoft.SqlTools.ServiceLayer/Localization/sr.resx

```diff
@@ -1535,6 +1535,10 @@
     <value>Cannot access the specified path on the server: {0}</value>
     <comment></comment>
   </data>
+  <data name="ConnInfoIsNull" xml:space="preserve">
+    <value>Connection Info is null</value>

```

Comment:
Let's avoid using terms like "null" in user-facing messages. Something more general like "Unable to find connection info for URI {0}" would be better I think.